modules/social_features/social_event/social_event.module

```diff
@@ -445,15 +482,15 @@ function social_event_social_user_account_header_account_links(array $context) {
 
   return [
     'my_events' => [
-      '#type' => 'link',
```

Comment:
@dmytriikaiun Looks like those changes not relevant and were added from code formatting of your IDE.